lib/lib.ml

```diff
@@ -237,6 +256,10 @@ module Conversion_ocamlnet (* : CONVERT *) = struct
   let update_both_filenames ?(ext="") ?(star=false) =
     update_filename ~ext:ext ~star:star
     << update_filename ~ext:ext ~star:true
+
+  (* TODO: Converts all attachments in an email, used by the
+     executable code, definition depends on that of a_copy_email *)
+  let full_convert_email _ _ = Error `DummyError
```

Comment:
This'll just be `print << acopy`, won't it?

Reply:
Yep, I just wasn't sure how Cormac would handle errors so I created a dummy function. In reality, I should just use `acopy` in `main.lib`, and that will be possible once I know the new signature it has.